backend/routes/api/spots.js

```diff
@@ -324,29 +324,37 @@ router.delete('/:id', requireAuth, async (req, res, next) => {
 );
 
 //delete a spot image
-// router.delete('/:id/images/:imageId', requireAuth, async (req, res, next) => {
-//     const loggedInUser = req.user.id;
-//     const spotId = +req.params.id;
-//     const spotByID = await Spot.findByPk(spotId);
-
-//     if(!spotByID){
-//         const err = new Error("Spot couldn't be found");
-//         err.status = 404;
-//         return next(err);
-//     }
-//     if(loggedInUser !== spotByID.ownerId){
-//         const err = new Error("Forbidden");
-//         err.status = 403;
-//         return next(err);
-//     }
-
-//     await spotByID.destroy();
-//     res.status(200);
-//     return res.json({
-//         message: "Successfully deleted",
-//         statusCode: 200
-//     });
-//   }
-// );
+router.delete('/:id/images/:imageId', requireAuth, async (req, res, next) => {
+    const loggedInUser = req.user.id;
+    const spotId = +req.params.id;
+    const imageId = +req.params.imageId;
+    const spotByID = await Spot.findByPk(spotId);
+
+    if(!spotByID){
+        const err = new Error("Spot couldn't be found");
+        err.status = 404;
+        return next(err);
+    }
+    if(loggedInUser !== spotByID.ownerId){
+        const err = new Error("Forbidden");
+        err.status = 403;
+        return next(err);
+    }
+
+    const imageByID = await SpotImage.findByPk(imageId);
```

Comment:
May have a work around in deleting an image of a review that doesn't belong to current user

example : 
- spot queried for by the id provided by the params may belong to the current user
- image queried for by the imageId provided by the params isnt associated with the spot queried for ( spotId on the image record doesn't match the spot id provided

Reply:
@E-F-III dded where clause to grab the image belongs to the review